runtime/util/thrinfo.c

```diff
@@ -287,7 +287,22 @@ getVMThreadStateHelper(J9VMThread *targetThread,
 						 */
 					}
 				}
-	
+
+#if JAVA_SPEC_VERSION >= 19
+				/* For a J9VMThread with a mounted virtual thread that is waiting on a
+				 * lock, the virtual thread object should be returned for the lock object
+				 * and the corresponding J9VMThread should be returned for the lock
+				 * owner. This behaviour matches the RI's behaviour for ThreadMXBean.
+				 * There is no explicit documentation. ThreadMXBean documentation
+				 * does not support VirtualThread. Hence, no lock related to VirtualThread
+				 * has been defined.
+				 */
+				if (((J9VMTHREAD_STATE_WAITING == vmstate) || (J9VMTHREAD_STATE_WAITING_TIMED == vmstate))
+				&& (NULL != targetThread->currentContinuation)) {
```

Comment:
should be on a new line.
```suggestion
				&& (NULL != targetThread->currentContinuation)
				) {
```